### PR TITLE
[ZH] Update `Ranking_Criteria`

### PR DESCRIPTION
**Pull request by YumeMuzi**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)
- Changes from other PRs are properly applied:
  - [x] https://github.com/ppy/osu-wiki/pull/6872
  - [x] https://github.com/ppy/osu-wiki/pull/6840
  - [x] IF/WHEN it's merged: https://github.com/ppy/osu-wiki/pull/6890

### CODE REVIEW

File: wiki/Ranking_Criteria/zh.md [hunk 1]
```diff
@@ -54,7 +49,7 @@ outdated_since: 7742320aaa249169bcb0536d8fc19dc7d32015fe
   - **小于 3:30，则最低难度不得高于 Normal。**
   - **在 3:30 到 4:15 之间，则最低难度不得高于 Hard。**
   - **在 4:15 到 5:00 之间，则最低难度不得高于 Insane。**
-  - **最高难度以下的难度可以使用包含休息时间的总游戏时长来替代掉血时间，作为度量标准而达到规定阈值。** 这不包含掉血时间在 30 秒以下的难度。一般由该游戏模式的 [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) 和 [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) 成员来判断谱面中作为最低难度的 Hard 或者 Insane 谱面是否合格。
+  - **最高难度以下的难度可以使用包含休息时间的总游戏时长来替代掉血时间，作为度量标准而达到规定阈值。** 这不包含掉血时间在 30 秒以下的难度。一般由该游戏模式的 [BN](/wiki/People/The_Team/Beatmap_Nominators) 和 [NAT](/wiki/People/The_Team/Nomination_Assessment_Team) 成员来判断谱面中作为最低难度的 Hard 或者 Insane 谱面是否合格。
```
Comment (RushFTK) [inline wiki/Ranking_Criteria/zh.md:52]:
```suggestion
  - **最高难度以下的难度可以使用包含休息时间的总游戏时长来替代掉血时间，作为度量标准而达到规定阈值。** 这不包含掉血时间在 30 秒以下的难度。一般由该游戏模式的 [谱面提名者](/wiki/People/The_Team/Beatmap_Nominators) (以下简称BN) 和 [提名评估团队](/wiki/People/The_Team/Nomination_Assessment_Team) (以下简称NAT) 成员来判断谱面中作为最低难度的 Hard 或者 Insane 谱面是否合格。
```

首次出现的专有名词缩写，建议保留原有全称。如果认为原有英文名称不恰当，可使用中文翻译表示。不过由于不常见，所以并不推荐。

**Review by RushFTK — DISMISSED** [review]
all looks fine for me (as a chinese native speaker). while a few point may discussion.

File: wiki/Ranking_Criteria/zh.md (hunk 1 above)
Comment (RushFTK) [reply wiki/Ranking_Criteria/zh.md:52]: 这段我看了确实只有这一处出现，所以也许你不修改也可以。

Comment (YumeMuzi) [reply wiki/Ranking_Criteria/zh.md:52]: 因为中文社区内很少用全称，而且在目前没有专用的名词解释下，用缩写也可以
Abbreviations are often used to refer to them in the osu! Chinese community.

**Comment by TicClick** [comment]
I've just submitted for review https://github.com/ppy/osu-wiki/pull/6870 with small improvements for the original article -- please make sure to apply changes from it **once it's finalized and merged**

**Review by RushFTK — CHANGES_REQUESTED** [review]
Others looks fine, but keep some terms like `tags` as it was or translate should be determined.
现在这个版本的翻译中，"标签"到底是主要使用中文，在首次使用中注释英文；或者主要使用英文，在首次使用中注释中文。

File: wiki/Ranking_Criteria/zh.md [hunk 2]
```diff
@@ -119,27 +114,30 @@ outdated_since: 7742320aaa249169bcb0536d8fc19dc7d32015fe
...
 #### 技术
 
-- **如果是作图者改编或翻唱了歌曲，则可以随意给它取适当的名字以表明这首歌是一个特殊版本。** 在这种情况下，为了让玩家能够轻易搜索到原歌曲，原歌曲的名字仍必须清楚地指示出来。
+- **如果是谱师改编或翻唱了歌曲，则可以按照谱师的想法给谱面取适当的名字以表明这首歌是一个特殊版本。** 在这种情况下，为了让玩家能够轻易搜索到原歌曲，仍必须清楚地标出原歌曲名。
 - **对于歌曲串烧或重编曲，应在 Tags 标签字段中填写所使用原歌曲的曲名和艺术家。** 这是为了保证玩家可以通过搜索同一关键字来找到与同一首歌相关的所有谱面，而不会得到很多不同的结果。
 - **若元数据中包含难以打出或难以辨认的 Unicode 字符，应在 Tags 标签字段中填写易于搜索的变形字段或者罗马音化字段。**
 - **如果歌曲来源同时有 Unicode 与罗马音化形式，应在 Tags 标签字段中填写之前未在 Source 字段中使用的形式。**
-- **歌曲风格与语言也应加入 Tags 字段中。** 这是为了让玩家能在游戏内也能像在网站上一样搜索到它。对于纯音乐，使用 `instrumental` 作为语言标记。这不包含语言或曲风不清晰的场合。如果适用于多个语言或曲风，则都应填写。
-- **Tags 还应加入相关艺术家、歌曲的另一个标题、或另一个艺术家名称的拼写、元数据字段中简写词汇的完整字段，以及任何能够帮助玩家找到该谱面的内容。**
+- **[歌曲风格与语言](/wiki/Beatmap/Genre_and_language)也应加入 Tags 字段中。** 这是为了让玩家能在游戏内也能像在网站上一样搜索到它。使用 `instrumental` 作为语言标记纯音乐类歌曲。且不应在语言或曲风不清晰时使用这个标记。如果歌曲适用于多个语言或曲风，则都应填写。
```
[the remark this answers is not in the document]
Comment (YumeMuzi) [reply wiki/Ranking_Criteria/zh.md:139]: 用英文 Tags 吧 等会改

File: wiki/Ranking_Criteria/zh.md [hunk 3]
```diff
@@ -158,22 +156,22 @@ outdated_since: 7742320aaa249169bcb0536d8fc19dc7d32015fe
...
-- **对于重编曲或翻唱，原始艺术家可以被用作艺术家字段，同时修改标题字段来指明这是一个重编曲。** 此标记应总是在括号内，并在描述词汇之后跟上重编曲艺术家。
+- **对于重编曲或翻唱，可以将原始艺术家填入艺术家字段，同时修改标题字段来指明这是一首重编歌曲。** 需要用括号括住修改的内容，并在描述词汇之后标注重编曲艺术家。
 
 ## 时间轴测定
 
 ### 规定
 
-- **必须使用[非继承时间点](/wiki/Client/Beatmap_editor/Timing#uninherited-timing-point)来准确地表述歌曲的拍子记号。** 如果错误的拍子记号持续超过一个小节，则必须在下一个强拍添加非继承时间点重置拍子记号。由于编辑器不支持 #/4 拍子记号，可以重置节拍器或编辑 `.osu` 文件。对于其他不支持的时间记号，参见[范例图表](/wiki/shared/timing/Timing_signature_reference_chart.png)或者查阅[这个指南](/wiki/Guides/Timing_Songs_With_8-Signatures)获取更多信息。
+- **必须使用[非继承时间点](/wiki/Client/Beatmap_editor/Timing#uninherited-timing-point)来准确地表述歌曲的拍子记号。** 如果错误的拍子记号持续超过一个小节，则必须在下一个强拍添加非继承时间点重置拍子记号。由于编辑器不支持 #/4 拍子记号，可以重置节拍器或编辑 `.osu` 文件。对于其他不支持的时间记号，参见[范例图表](/wiki/shared/timing/Timing_signature_reference_chart.png)或者查阅[八分拍歌曲测定](/wiki/Guides/Timing_Songs_With_8-Signatures)获取更多信息。
 - **谱面必须测定出精准的时间轴。** 这意味着 [BPM](/wiki/Beatmapping/Beats_per_minute) 和[偏移量](/wiki/Beatmapping/Offset)需要完全贴合歌曲。无规律的变奏歌曲也许不可能完美地测定，那么则应在不影响谱面游玩的情况下尽量精确测定。如果打算采用休息段或者转盘覆盖变奏段，则这个段落的复杂节奏测定可以忽略掉。
```
[the remark this answers is not in the document]
Comment (YumeMuzi) [reply wiki/Ranking_Criteria/zh.md:166]: #6870 已经得出结果，等会改

**Review by RushFTK — APPROVED** [review]
looks fine for a Chinese native speaker.

**Comment by Walavouchey** [comment]
let's not balloon the scope too much for this pr. https://github.com/ppy/osu-wiki/pull/6873 may take a bit to get merged

File: wiki/Ranking_Criteria/zh.md [hunk 4]
```diff
@@ -1,17 +1,23 @@
...
+## 工具
+
+非常建议您使用 [Mapset Verifier (MV)](https://github.com/Naxesss/MapsetVerifier) 来帮助检查谱面是否符合以下这些规则。老旧的稳定版 (stable) 客户端内的 [AiMod](/wiki/Client/Beatmap_editor/AiMod) 工具已经过时且不再维护。同时，新版 lazer 客户端内的谱面检查工具仍在[开发中](https://github.com/ppy/osu/issues/12091#issuecomment-878760791)。
+
+使用前需注意：
+
+- 工具不能代替阅读此页面或人工检查的必要操作。
+- 请保持自己的判断，不要盲目地遵循工具指出的建议。
+- 工具能帮助你检查谱面，而不能代替你检查谱面。
+
```
Comment (Walavouchey) [inline wiki/Ranking_Criteria/zh.md:20]: excess line

```suggestion
```

**Comment by Walavouchey** [comment]
@RushFTK sorry, mind taking one more look for latest change ([a055904](https://github.com/ppy/osu-wiki/pull/6848/commits/a055904996fb31808b1c9f695a21b848b0e0d600))?

**Comment by TicClick** [comment]
fair, I didn't follow it closely

File: wiki/Ranking_Criteria/zh.md [hunk 5]
```diff
@@ -2,12 +2,23 @@
...
-**Ranking Criteria** 列出了让[谱面](/wiki/Beatmap)通过[谱面 Ranking 程序](/wiki/Beatmap_ranking_procedure)时须遵守的[规则和准则](#一般术语)
+**Ranking Criteria** 列出了让[谱面](/wiki/Beatmap)通过[谱面 Ranking 程序](/wiki/Beatmap_ranking_procedure)时须遵守的[规定和准则](#一般术语)
 
 可在 [Ranking Criteria 论坛](https://osu.ppy.sh/community/forums/87)中提出和讨论对本文的更正。并参照 *[如何建议更改 Ranking 准则](https://osu.ppy.sh/community/forums/topics/720532)* 来提供贡献。注意，若仅对本文中的遣词、语法、组句进行改变，而不改变其原意，则无需发帖至论坛。
 
 **注意[摸图和作图的行为准则](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping)、[歌曲内容规定](/wiki/Rules/Song_Content_Rules)是 Ranking Criteria 的一部分，适用于 osu! 中所有的游戏模式。**
 
+## 工具
+
+非常建议您使用 [Mapset Verifier (MV)](https://github.com/Naxesss/MapsetVerifier) 来帮助检查谱面是否符合以下这些规则。老旧的稳定版 (stable) 客户端内的 [AiMod](/wiki/Client/Beatmap_editor/AiMod) 工具已经过时且不再维护。同时，新版 lazer 客户端内的谱面检查工具仍在[开发中](https://github.com/ppy/osu/issues/12091#issuecomment-878760791)。
```
Comment (RushFTK) [inline wiki/Ranking_Criteria/zh.md:13]:
````suggestion
非常建议您使用 [Mapset Verifier (MV)](https://github.com/Naxesss/MapsetVerifier) 来帮助检查谱面是否符合本文所述的规则。传统客户端 (stable 版) 内的 [AiMod](/wiki/Client/Beatmap_editor/AiMod) 工具已经过时且不再维护。同时，新版客户端( lazer 版) 内的谱面检查工具仍在[开发中](https://github.com/ppy/osu/issues/12091#issuecomment-878760791)。
````

File: wiki/Ranking_Criteria/zh.md [hunk 6]
```diff
@@ -2,12 +2,23 @@
...
+
+使用前需注意：
+
+- 工具不能代替阅读此页面或人工检查的必要操作。
```
Comment (RushFTK) [inline wiki/Ranking_Criteria/zh.md:17]:
````suggestion
- 工具不能代替阅读此页面或人工检查的必要性。
````

File: wiki/Ranking_Criteria/zh.md [hunk 7]
```diff
@@ -2,12 +2,23 @@
...
+- 请保持自己的判断，不要盲目地遵循工具指出的建议。
+- 工具能帮助你检查谱面，而不能代替你检查谱面。
```
Comment (RushFTK) [inline wiki/Ranking_Criteria/zh.md:19]:
````suggestion
- 工具是用于帮助你检查谱面，而不是代替你检查谱面。
````

File: wiki/Ranking_Criteria/zh.md (hunk 6 above)
Comment (RushFTK) [reply wiki/Ranking_Criteria/zh.md:17]: "阅读此页"与词语"操作"不匹配。

File: wiki/Ranking_Criteria/zh.md (hunk 7 above)
Comment (RushFTK) [reply wiki/Ranking_Criteria/zh.md:19]: 原文含义毕竟是促使玩家自己审一遍铺面，主要强调谱师的主观用法而不是AiMod的性能。能其实还是能的（）

**Review by RushFTK — CHANGES_REQUESTED** [review]
No problem @Walavouchey , here is the review as requested.
in my opinion, the second review is necessary to change, while others are opinion.

**Review by RushFTK — APPROVED** [review]
Looks fine for native speakers now.

**Comment by Walavouchey** [comment]
it looks like all the rc articles have the same issue as in https://github.com/ppy/osu-wiki/pull/6851#discussion_r808927887. i'll update all of them in a separate pr

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**